cms/static/js/spec/views/pages/course_outline_spec.js

```diff
@@ -1339,6 +1346,96 @@ describe('CourseOutlinePage', function() {
+                    createMockSubsectionJSON({
+                        has_changes: true,
+                        is_time_limited: true,
+                        is_practice_exam: true,
+                        is_proctored_exam: true,
+                        default_time_limit_minutes: 150,
+                        supports_onboarding: true,
+                    }, [
+                    ])
+                ])
+            ]);
+
+            createCourseOutlinePage(this, mockCourseWithSpecialExamJSON, false);
+            outlinePage.$('.outline-subsection .configure-button').click();
+            setEditModalValues('7/9/2014', '7/10/2014', 'Lab');
+            selectVisibilitySettings();
+            setContentVisibility('staff_only');
+            selectAdvancedSettings();
+            selectOnboardingExam('00:30');
+
+            // time limit should be visible, review rules should be hidden
+            checkOptionFieldVisibility(true, false);
+
+            $('.wrapper-modal-window .action-save').click();
+        });
+
+        it('does not show practice exam option when course supports onboarding', function() {
+            var mockCourseWithSpecialExamJSON = createMockCourseJSON({}, [
+                createMockSectionJSON({
+                    has_changes: true,
+                    enable_proctored_exams: true,
+                    enable_timed_exams: true
+
+                }, [
+                    createMockSubsectionJSON({
+                        has_changes: true,
+                        is_time_limited: true,
+                        is_practice_exam: true,
+                        is_proctored_exam: true,
+                        default_time_limit_minutes: 150,
+                        supports_onboarding: true,
+                    }, [
+                    ])
+                ])
+            ]);
+
+            createCourseOutlinePage(this, mockCourseWithSpecialExamJSON, false);
+            outlinePage.$('.outline-subsection .configure-button').click();
+            selectAdvancedSettings();
+            expect($('input.practice_exam')).not.toExist();
+            expect($('input.onboarding_exam')).toExist();
+            expect($('.field-time-limit input').val()).toBe('02:30');
+        });
+
+        it('does not show onboarding exam option when course does not support onboarding', function() {
+            var mockCourseWithSpecialExamJSON = createMockCourseJSON({}, [
+                createMockSectionJSON({
+                    has_changes: true,
+                    enable_proctored_exams: true,
+                    enable_timed_exams: true
+
+                }, [
+                    createMockSubsectionJSON({
+                        has_changes: true,
+                        is_time_limited: true,
+                        is_practice_exam: true,
+                        is_proctored_exam: true,
+                        default_time_limit_minutes: 150,
+                        supports_onboarding: true,
```

Comment:
Looking closely at the description of this test (`when course does not support onboarding`), shouldn't this be false?